### PR TITLE
[fix issue #1180] Too many update operation violates with apiserver concurrency...

### LINKED ISSUE #1180: [BUG] Too many update operation violates with apiserver concurrency control after refactoring
**What is your environment(Kubernetes version, Fluid version, etc.)**

**Describe the bug**

docker image: v0.7.0-79802b3

```
2021-11-23T16:08:21.586+0800	INFO	utils	utils/errors.go:12	Retry later when update operation violates with apiserver concurrency control.	{"error": "Operation cannot be fulfilled on statefulsets.apps \"hbase-worker\": the object has been modified; please apply your changes to the latest version and try again", "name": "hbase", "namespace": "default"}
2021-11-23T16:08:21.606+0800	INFO	utils	utils/errors.go:12	Retry later when update operation violates with apiserver concurrency control.	{"error": "Operation cannot be fulfilled on statefulsets.apps \"hbase-worker\": the object has been modified; please apply your changes to the latest version and try again", "name": "hbase", "namespace": "default"}
2021-11-23T16:08:21.669+0800	INFO	utils	utils/errors.go:12	Retry later when update operation violates with apiserver concurrency control.	{"error": "Operation cannot be fulfilled on statefulsets.apps \"hbase-worker\": the object has been modified; please apply your changes to the latest version and try again", "name": "hbase", "namespace": "default"}
2021-11-23T16:08:21.941+0800	INFO	utils	utils/errors.go:12	Retry later when update operation violates with apiserver concurrency control.	{"error": "Operation cannot be fulfilled on statefulsets.apps \"hbase-worker\": the object has been modified; please apply your changes to the latest version and try again", "name": "hbase", "namespace": "default"}
2021-11-23T16:08:21.941+0800	INFO	utils	utils/errors.go:12	Retry later when update operation violates with apiserver concurrency control.	{"error": "Operation cannot be fulfilled on statefulsets.apps \"hbase-worker\": the object has been modified; please apply your changes to the latest version and try again", "name": "hbase", "namespace": "default"}
2021-11-23T16:08:21.941+0800	INFO	utils	utils/errors.go:12	Retry later when update operation violates with apiserver concurrency control.	{"error": "Operation cannot be fulfilled on statefulsets.apps \"hbase-worker\": the object has been modified; please apply your changes to the latest version and try again", "name": "hbase", "namesp
```

**What you expect to happen:**

**How to reproduce it**

**Additional Information**

### PR DESCRIPTION
**Pull request by daomin885**
Signed-off-by: 道民 <jiebin.cjb@alibaba-inc.com>

### Ⅱ. Does this pull request fix one issue?
fixes #1180

### Ⅴ. Special notes for reviews

### CODE REVIEW

**Review by cheyang — CHANGES_REQUESTED** [review]

File: pkg/ddc/alluxio/worker.go [hunk 1]
```diff
@@ -29,26 +29,19 @@ import (
...
 		runtime, err := e.getRuntime()
 		if err != nil {
 			return err
 		}
 		runtimeToUpdate := runtime.DeepCopy()
 		err = e.Helper.SetupWorkers(runtimeToUpdate, runtimeToUpdate.Status, workers)
 		if err != nil {
-			_ = utils.LoggingErrorExceptConflict(e.Log, err, "Failed to setup worker",
-				types.NamespacedName{Namespace: e.namespace, Name: e.name})
+			_ = utils.LoggingErrorExceptConflict(e.Log, err, "Failed to check worker ready", types.NamespacedName{Namespace: e.namespace, Name: e.name})
```
Comment (cheyang) [inline pkg/ddc/alluxio/worker.go:44]: I think you don't need to call `utils.LoggingErrorExceptConflict` in line 44.

File: pkg/ddc/alluxio/health_check.go [hunk 2]
```diff
@@ -211,7 +210,7 @@ func (e *AlluxioEngine) checkWorkersHealthy() (err error) {
 		if !reflect.DeepEqual(runtime.Status, runtimeToUpdate.Status) {
 			updateErr := e.Client.Status().Update(context.TODO(), runtimeToUpdate)
 			if updateErr != nil {
-				e.Log.Error(updateErr, "Failed to update the runtime")
+				_ = utils.LoggingErrorExceptConflict(e.Log, updateErr, "Failed to update runtime status", types.NamespacedName{Namespace: e.namespace, Name: e.name})
```
[the remark this answers is not in the document]
Comment (daomin885) [reply pkg/ddc/alluxio/health_check.go]: good suggestion!

File: pkg/ddc/alluxio/worker.go [hunk 3]
```diff
@@ -96,8 +86,7 @@ func (e *AlluxioEngine) CheckWorkersReady() (ready bool, err error) {
 		runtimeToUpdate := runtime.DeepCopy()
 		ready, err = e.Helper.CheckWorkersReady(runtimeToUpdate, runtimeToUpdate.Status, workers)
 		if err != nil {
-			_ = utils.LoggingErrorExceptConflict(e.Log, err, "Failed to setup worker",
-				types.NamespacedName{Namespace: e.namespace, Name: e.name})
+			_ = utils.LoggingErrorExceptConflict(e.Log, err, "Failed to check worker ready", types.NamespacedName{Namespace: e.namespace, Name: e.name})
```
Comment (cheyang) [inline pkg/ddc/alluxio/worker.go:89]: I suggest to remove 

```
if err != nil {
			_ = utils.LoggingErrorExceptConflict(e.Log, err, "Failed to check worker ready", types.NamespacedName{Namespace: e.namespace, Name: e.name})
```

**Review by cheyang — CHANGES_REQUESTED** [review]

File: pkg/ddc/alluxio/worker.go (hunk 3 above)
Comment (daomin885) [reply pkg/ddc/alluxio/worker.go:96]: thanks, i will remove it~

**Review by cheyang — APPROVED** [review]

**Pull request merged**